pkg/shed/db.go

```diff
@@ -81,6 +84,7 @@ func NewDB(path string, metricsPrefix string) (db *DB, err error) {
 func (db *DB) Put(key []byte, value []byte) (err error) {
 	err = db.ldb.Put(key, value, nil)
 	if err != nil {
+		db.logger.Debugf("failed to insert in to DB", "err", err)
```

Comment:
This usage of Debugf is not correct.

Reply:
@jmozah could you just fix this, and we can merge?

Reply:
@janos oops... forgot this one...Dne it now...